### ISSUE TITLE
Accessibility: Child items/dropdown menu items inside a view fails for  WCAG 2.2 , SC 4.1.2 Name, Role, Value (Level A)

### ISSUE THREAD

**Issue by RushikeshTrivedi** [microsoft/vscode#321958]
Description:
The ARIA attributes "aria-label" are not valid for the element <div> with implicit ARIA role "generic"

VS Code Version: 1.119.0
OS Version: Windows 11

**Steps to Reproduce:**

1. Open VSCODE
2. Select Extension
3. Run accessibility checker on it

**Screenshots for views showing this error:**

Outline default view showing same issue:

<img width="3762" height="1348" alt="Image" src="https://github.com/user-attachments/assets/087e6533-f2f2-4f35-8282-86f7f4d0b117" />

<img width="3793" height="1316" alt="Image" src="https://github.com/user-attachments/assets/1fef8cc9-f20c-469f-88bf-1856725e98c3" />

**Rationale:**
This requirement ensures that Assistive Technologies (AT) can gather information about, activate, set, and receive status of all user interface components.

**Note:** This requirement is primarily for developers who develop or script their own custom user interface components. Standard HTML controls already meet this requirement when used according to specification.

Custom interactive user interface components often do not expose proper role, state, and property information needed for assistive technologies. As a result, people with disabilities have difficulty identifying and interacting with the components. Common examples are custom menus, trees, data grids, accordions, and dialog boxes. Ensuring proper markup and keyboard operation enables people with disabilities to identify and interact with the custom components in the same manner they identify and interact with standard components.
